pkg/util/util.go

```diff
@@ -169,3 +170,16 @@ func PrettyPrintStruct(obj interface{}) string {
 	}
 	return string(str)
 }
+
+// MarshalNoHTMLEscape is nearly same as json.Marshal but does NOT HTLM-escape <, > or &
+// However it does add a newline char at the end (as done by json.Encoder.Encode)
+func MarshalNoHTMLEscape(v interface{}) ([]byte, error) {
```

Comment:
Maybe include JSON in the function name to specify that this method Mashals json data.

Reply:
as discussed per slack, leaving it like that for now